data/datasets.yml

```diff
@@ -788,3 +745,455 @@ JCA-All: # All JCA misuses.
   - jmrtd.67.2
   - pawotag.82.1
   - tab-apps.62.1
+
+
+JCA-Param-All: # All parametric cryptographic misuses identified in the set.
```

Comment:
Please mention in a comment that not all of these are compilable.

Reply:
I have clarified this in commit 43f5c93ab1a8ec982197dc2f9846d815cb987b07.